pkg/query-service/app/traces/v3/query_builder_test.go

```diff
@@ -419,7 +419,7 @@ var testOrderBy = []struct {
 			},
 		},
 		Tags:   []string{},
-		Result: []string{"name asc", "bytes asc", "stringTagMap['response_time'] desc"},
+		Result: []string{"`name` asc", "`bytes` asc", "`stringTagMap['response_time']` desc"},
```

Comment:
This query will not run. The "\`stringTagMap['response_time']\`" will be interpreted as a literal column name instead of the `stringTagMap['response_time']`, but the column name doesn't exist

Reply:
Thanks for pointing out!
Have fixed this by only using backticks on columns in orderBy